packages/lucia/src/middleware/index.ts

```diff
@@ -115,7 +115,7 @@ export const sveltekit = (): Middleware<[SvelteKitRequestEvent]> => {
 			request: event.request,
 			sessionCookie: event.cookies.get(sessionCookieName) ?? null,
 			setCookie: (cookie) => {
-				event.cookies.set(cookie.name, cookie.value, cookie.attributes);
+				event.cookies.set(cookie.name, cookie.value, { path: '/', ...cookie.attributes });
```

Comment:
Yeah I'd set the default to `""` to be consistent
```suggestion
				event.cookies.set(cookie.name, cookie.value, { path: '', ...cookie.attributes });
```

Reply:
Gotcha, good call.